components/stripe/actions/retrieve-checkout-line-item-product/retrieve-checkout-line-item-product.js

```diff
@@ -0,0 +1,23 @@
+const stripe = require("../../stripe.app.js");
+
+export default {
+  name: "Retrieve Product",
+  description: "Retrieve a product by ID. [See the docs](https://stripe.com/docs/api/products/retrieve)",
+  key: "stripe-retrieve-product",
```

Comment:
```suggestion
  key: "stripe-retrieve-checkout-line-item-product",
```

Reply:
Ahhh...I see why you've suggested this - the file is stilled called retrieve-checkout-line-item-products. When I first made it, that's what I thought it was going to do, but it's evolved into just retrieving a product. I'm going to rename the file and folder it's in instead. Agreed?